Build failure: octavePackages.ltfat (darwin)

## Steps To Reproduce

`nix-shell -p octavePackages.ltfat`

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

```
Sourcing write-required-octave-packages-hook.sh
Using writeRequiredOctavePackagesPhase
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/4vmpg7vfzwx1xz2vi2m118m177nnjs74-ltfat-2.6.0-of.tar.gz
source root is ltfat
setting SOURCE_DATE_EPOCH to timestamp 1694210710 of file ltfat/inst/demos/demo_dgt_parametrize.m
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
ar: [0;1;35mwarning: [0mcreating build/libltfat.a
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
ld: library not found for -loctinterp
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_cellcoef2tf.oct] Error 1
make[1]: *** Waiting for unfinished jobs....
ld: library not found for -loctinterp
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_dwilt.oct] Error 1
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_atrousfilterbank_td.oct] Error 1
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_col2diag.oct] Error 1
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_dwiltiii.oct] Error 1
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_dst.oct] Error 1
make[1]: *** [Makefile_unix:50: comp_chirpzt.oct] Error 1
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
make[1]: *** [Makefile_unix:50: comp_filterbank_fftbl.oct] Error 1
make[1]: *** [Makefile_unix:50: comp_dct.oct] Error 1
make[1]: *** [Makefile_unix:50: comp_filterbank_fft.oct] Error 1
ld: library not found for -loctinterp
clang-16: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
make[1]: *** [Makefile_unix:50: comp_filterbank_td.oct] Error 1
make: *** [Makefile:41: oct] Error 2
make: Entering directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/src'
# Compile the backend lib, no dependency is resolved here since we compile
make -C ../thirdparty/Playrec -f Makefile_unixoct
make -C ../thirdparty/polyboolclipper -f Makefile_unixoct
# a static library
make[1]: Entering directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/thirdparty/Playrec'
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -Wall -mex mex_dll_core.c pa_dll_playrec.c ltfatresample.c \
             -L. -lportaudio -I. -DIS_OCTAVE -DHAVE_PORTAUDIO	-o playrec.mex
make -f Makefile_libltfat build/libltfat.a OPTCFLAGS=-DLTFAT_LARGEARRAYS
make[1]: Entering directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/thirdparty/polyboolclipper'
make[1]: Entering directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/src'
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/memalloc.c -o obj/libltfat/common/dmemalloc.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/error.c -o obj/libltfat/common/derror.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/version.c -o obj/libltfat/common/dversion.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/argchecks.c -o obj/libltfat/common/dargchecks.o 
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -Wall -mex polyboolmex.cpp clipper.cpp
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/dgtwrapper_typeconstant.c -o obj/libltfat/common/ddgtwrapper_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/dgtrealmp_typeconstant.c -o obj/libltfat/common/ddgtrealmp_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/reassign_typeconstant.c -o obj/libltfat/common/dreassign_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/wavelets_typeconstant.c -o obj/libltfat/common/dwavelets_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/integer_manip.c -o obj/libltfat/common/dinteger_manip.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/firwin_typeconstant.c -o obj/libltfat/common/dfirwin_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt.c -o obj/libltfat/double/dgt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtreal_fb.c -o obj/libltfat/double/dgtreal_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_multi.c -o obj/libltfat/double/dgt_multi.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_ola.c -o obj/libltfat/double/dgt_ola.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_shear.c -o obj/libltfat/double/dgt_shear.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtreal_long.c -o obj/libltfat/double/dgtreal_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dwilt.c -o obj/libltfat/double/dwilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idwilt.c -o obj/libltfat/double/idwilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/wmdct.c -o obj/libltfat/double/wmdct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/iwmdct.c -o obj/libltfat/double/iwmdct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/filterbank.c -o obj/libltfat/double/filterbank.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ifilterbank.c -o obj/libltfat/double/ifilterbank.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/heapint.c -o obj/libltfat/double/heapint.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/heap.c -o obj/libltfat/double/heap.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/wfacreal.c -o obj/libltfat/double/wfacreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idgtreal_long.c -o obj/libltfat/double/idgtreal_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idgtreal_fb.c -o obj/libltfat/double/idgtreal_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/iwfacreal.c -o obj/libltfat/double/iwfacreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/pfilt.c -o obj/libltfat/double/pfilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/reassign_ti.c -o obj/libltfat/double/reassign_ti.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/windows.c -o obj/libltfat/double/windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_shearola.c -o obj/libltfat/double/dgt_shearola.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/utils.c -o obj/libltfat/double/utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/rtdgtreal.c -o obj/libltfat/double/rtdgtreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/circularbuf.c -o obj/libltfat/double/circularbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/slicingbuf.c -o obj/libltfat/double/slicingbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealwrapper.c -o obj/libltfat/double/dgtrealwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealmp.c -o obj/libltfat/double/dgtrealmp.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealmp_parbuf.c -o obj/libltfat/double/dgtrealmp_parbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealmp_kernel.c -o obj/libltfat/double/dgtrealmp_kernel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealmp_guts.c -o obj/libltfat/double/dgtrealmp_guts.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/maxtree.c -o obj/libltfat/double/maxtree.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/slidgtrealmp.c -o obj/libltfat/double/slidgtrealmp.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/filterbankphaseret.c -o obj/libltfat/double/filterbankphaseret.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/fbheapint.c -o obj/libltfat/double/fbheapint.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/fftw_wrappers.c -o obj/libltfat/double/fftw_wrappers.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ltfat_blaslapack.c -o obj/libltfat/double/ltfat_blaslapack.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabdual_fac.c -o obj/libltfat/double/gabdual_fac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabtight_fac.c -o obj/libltfat/double/gabtight_fac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ci_utils.c -o obj/libltfat/double/ci_utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ci_windows.c -o obj/libltfat/double/ci_windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/spread.c -o obj/libltfat/double/spread.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/wavelets.c -o obj/libltfat/double/wavelets.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/goertzel.c -o obj/libltfat/double/goertzel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/reassign.c -o obj/libltfat/double/reassign.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabdual_painless.c -o obj/libltfat/double/gabdual_painless.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/wfac.c -o obj/libltfat/double/wfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/iwfac.c -o obj/libltfat/double/iwfac.o
make[1]: Leaving directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/thirdparty/Playrec'
mv ../thirdparty/Playrec/playrec.mex .
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_long.c -o obj/libltfat/double/dgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idgt_long.c -o obj/libltfat/double/idgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_fb.c -o obj/libltfat/double/dgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idgt_fb.c -o obj/libltfat/double/idgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ci_memalloc.c -o obj/libltfat/double/ci_memalloc.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtwrapper.c -o obj/libltfat/double/dgtwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dct.c -o obj/libltfat/double/dct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dst.c -o obj/libltfat/double/dst.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabdual.c -o obj/libltfat/double/gabdual.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabtight.c -o obj/libltfat/double/gabtight.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_utils.c -o obj/libltfat/complexdouble/ci_utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_windows.c -o obj/libltfat/complexdouble/ci_windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/spread.c -o obj/libltfat/complexdouble/spread.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/wavelets.c -o obj/libltfat/complexdouble/wavelets.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/goertzel.c -o obj/libltfat/complexdouble/goertzel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/reassign.c -o obj/libltfat/complexdouble/reassign.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabdual_painless.c -o obj/libltfat/complexdouble/gabdual_painless.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/wfac.c -o obj/libltfat/complexdouble/wfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/iwfac.c -o obj/libltfat/complexdouble/iwfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgt_long.c -o obj/libltfat/complexdouble/dgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/idgt_long.c -o obj/libltfat/complexdouble/idgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgt_fb.c -o obj/libltfat/complexdouble/dgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/idgt_fb.c -o obj/libltfat/complexdouble/idgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_memalloc.c -o obj/libltfat/complexdouble/ci_memalloc.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgtwrapper.c -o obj/libltfat/complexdouble/dgtwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dct.c -o obj/libltfat/complexdouble/dct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dst.c -o obj/libltfat/complexdouble/dst.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabdual.c -o obj/libltfat/complexdouble/gabdual.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabtight.c -o obj/libltfat/complexdouble/gabtight.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt.c -o obj/libltfat/single/dgt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtreal_fb.c -o obj/libltfat/single/dgtreal_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_multi.c -o obj/libltfat/single/dgt_multi.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_ola.c -o obj/libltfat/single/dgt_ola.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_shear.c -o obj/libltfat/single/dgt_shear.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtreal_long.c -o obj/libltfat/single/dgtreal_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dwilt.c -o obj/libltfat/single/dwilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idwilt.c -o obj/libltfat/single/idwilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/wmdct.c -o obj/libltfat/single/wmdct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/iwmdct.c -o obj/libltfat/single/iwmdct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/filterbank.c -o obj/libltfat/single/filterbank.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ifilterbank.c -o obj/libltfat/single/ifilterbank.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/heapint.c -o obj/libltfat/single/heapint.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/heap.c -o obj/libltfat/single/heap.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/wfacreal.c -o obj/libltfat/single/wfacreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idgtreal_long.c -o obj/libltfat/single/idgtreal_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idgtreal_fb.c -o obj/libltfat/single/idgtreal_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/iwfacreal.c -o obj/libltfat/single/iwfacreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/pfilt.c -o obj/libltfat/single/pfilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/reassign_ti.c -o obj/libltfat/single/reassign_ti.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/windows.c -o obj/libltfat/single/windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_shearola.c -o obj/libltfat/single/dgt_shearola.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/utils.c -o obj/libltfat/single/utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/rtdgtreal.c -o obj/libltfat/single/rtdgtreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/circularbuf.c -o obj/libltfat/single/circularbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/slicingbuf.c -o obj/libltfat/single/slicingbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealwrapper.c -o obj/libltfat/single/dgtrealwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealmp.c -o obj/libltfat/single/dgtrealmp.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealmp_parbuf.c -o obj/libltfat/single/dgtrealmp_parbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealmp_kernel.c -o obj/libltfat/single/dgtrealmp_kernel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealmp_guts.c -o obj/libltfat/single/dgtrealmp_guts.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/maxtree.c -o obj/libltfat/single/maxtree.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/slidgtrealmp.c -o obj/libltfat/single/slidgtrealmp.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/filterbankphaseret.c -o obj/libltfat/single/filterbankphaseret.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/fbheapint.c -o obj/libltfat/single/fbheapint.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/fftw_wrappers.c -o obj/libltfat/single/fftw_wrappers.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ltfat_blaslapack.c -o obj/libltfat/single/ltfat_blaslapack.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabdual_fac.c -o obj/libltfat/single/gabdual_fac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabtight_fac.c -o obj/libltfat/single/gabtight_fac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ci_utils.c -o obj/libltfat/single/ci_utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ci_windows.c -o obj/libltfat/single/ci_windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/spread.c -o obj/libltfat/single/spread.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/wavelets.c -o obj/libltfat/single/wavelets.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/goertzel.c -o obj/libltfat/single/goertzel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/reassign.c -o obj/libltfat/single/reassign.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabdual_painless.c -o obj/libltfat/single/gabdual_painless.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/wfac.c -o obj/libltfat/single/wfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/iwfac.c -o obj/libltfat/single/iwfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_long.c -o obj/libltfat/single/dgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idgt_long.c -o obj/libltfat/single/idgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_fb.c -o obj/libltfat/single/dgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idgt_fb.c -o obj/libltfat/single/idgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ci_memalloc.c -o obj/libltfat/single/ci_memalloc.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtwrapper.c -o obj/libltfat/single/dgtwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dct.c -o obj/libltfat/single/dct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dst.c -o obj/libltfat/single/dst.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabdual.c -o obj/libltfat/single/gabdual.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabtight.c -o obj/libltfat/single/gabtight.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_utils.c -o obj/libltfat/complexsingle/ci_utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_windows.c -o obj/libltfat/complexsingle/ci_windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/spread.c -o obj/libltfat/complexsingle/spread.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/wavelets.c -o obj/libltfat/complexsingle/wavelets.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/goertzel.c -o obj/libltfat/complexsingle/goertzel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/reassign.c -o obj/libltfat/complexsingle/reassign.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabdual_painless.c -o obj/libltfat/complexsingle/gabdual_painless.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/wfac.c -o obj/libltfat/complexsingle/wfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/iwfac.c -o obj/libltfat/complexsingle/iwfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgt_long.c -o obj/libltfat/complexsingle/dgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/idgt_long.c -o obj/libltfat/complexsingle/idgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgt_fb.c -o obj/libltfat/complexsingle/dgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/idgt_fb.c -o obj/libltfat/complexsingle/idgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_memalloc.c -o obj/libltfat/complexsingle/ci_memalloc.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgtwrapper.c -o obj/libltfat/complexsingle/dgtwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dct.c -o obj/libltfat/complexsingle/dct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dst.c -o obj/libltfat/complexsingle/dst.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabdual.c -o obj/libltfat/complexsingle/gabdual.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabtight.c -o obj/libltfat/complexsingle/gabtight.o
ar rv build/libltfat.a obj/libltfat/common/dmemalloc.o obj/libltfat/common/derror.o obj/libltfat/common/dversion.o obj/libltfat/common/dargchecks.o obj/libltfat/common/ddgtwrapper_typeconstant.o obj/libltfat/common/ddgtrealmp_typeconstant.o obj/libltfat/common/dreassign_typeconstant.o obj/libltfat/common/dwavelets_typeconstant.o obj/libltfat/common/dinteger_manip.o obj/libltfat/common/dfirwin_typeconstant.o obj/libltfat/double/dgt.o obj/libltfat/double/dgtreal_fb.o obj/libltfat/double/dgt_multi.o obj/libltfat/double/dgt_ola.o obj/libltfat/double/dgt_shear.o obj/libltfat/double/dgtreal_long.o obj/libltfat/double/dwilt.o obj/libltfat/double/idwilt.o obj/libltfat/double/wmdct.o obj/libltfat/double/iwmdct.o obj/libltfat/double/filterbank.o obj/libltfat/double/ifilterbank.o obj/libltfat/double/heapint.o obj/libltfat/double/heap.o obj/libltfat/double/wfacreal.o obj/libltfat/double/idgtreal_long.o obj/libltfat/double/idgtreal_fb.o obj/libltfat/double/iwfacreal.o obj/libltfat/double/pfilt.o obj/libltfat/double/reassign_ti.o obj/libltfat/double/windows.o obj/libltfat/double/dgt_shearola.o obj/libltfat/double/utils.o obj/libltfat/double/rtdgtreal.o obj/libltfat/double/circularbuf.o obj/libltfat/double/slicingbuf.o obj/libltfat/double/dgtrealwrapper.o obj/libltfat/double/dgtrealmp.o obj/libltfat/double/dgtrealmp_parbuf.o obj/libltfat/double/dgtrealmp_kernel.o obj/libltfat/double/dgtrealmp_guts.o obj/libltfat/double/maxtree.o obj/libltfat/double/slidgtrealmp.o obj/libltfat/double/filterbankphaseret.o obj/libltfat/double/fbheapint.o obj/libltfat/double/fftw_wrappers.o obj/libltfat/double/ltfat_blaslapack.o obj/libltfat/double/gabdual_fac.o obj/libltfat/double/gabtight_fac.o obj/libltfat/double/ci_utils.o obj/libltfat/double/ci_windows.o obj/libltfat/double/spread.o obj/libltfat/double/wavelets.o obj/libltfat/double/goertzel.o obj/libltfat/double/reassign.o obj/libltfat/double/gabdual_painless.o obj/libltfat/double/wfac.o obj/libltfat/double/iwfac.o obj/libltfat/double/dgt_long.o obj/libltfat/double/idgt_long.o obj/libltfat/double/dgt_fb.o obj/libltfat/double/idgt_fb.o obj/libltfat/double/ci_memalloc.o obj/libltfat/double/dgtwrapper.o obj/libltfat/double/dct.o obj/libltfat/double/dst.o obj/libltfat/double/gabdual.o obj/libltfat/double/gabtight.o obj/libltfat/complexdouble/ci_utils.o obj/libltfat/complexdouble/ci_windows.o obj/libltfat/complexdouble/spread.o obj/libltfat/complexdouble/wavelets.o obj/libltfat/complexdouble/goertzel.o obj/libltfat/complexdouble/reassign.o obj/libltfat/complexdouble/gabdual_painless.o obj/libltfat/complexdouble/wfac.o obj/libltfat/complexdouble/iwfac.o obj/libltfat/complexdouble/dgt_long.o obj/libltfat/complexdouble/idgt_long.o obj/libltfat/complexdouble/dgt_fb.o obj/libltfat/complexdouble/idgt_fb.o obj/libltfat/complexdouble/ci_memalloc.o obj/libltfat/complexdouble/dgtwrapper.o obj/libltfat/complexdouble/dct.o obj/libltfat/complexdouble/dst.o obj/libltfat/complexdouble/gabdual.o obj/libltfat/complexdouble/gabtight.o obj/libltfat/single/dgt.o obj/libltfat/single/dgtreal_fb.o obj/libltfat/single/dgt_multi.o obj/libltfat/single/dgt_ola.o obj/libltfat/single/dgt_shear.o obj/libltfat/single/dgtreal_long.o obj/libltfat/single/dwilt.o obj/libltfat/single/idwilt.o obj/libltfat/single/wmdct.o obj/libltfat/single/iwmdct.o obj/libltfat/single/filterbank.o obj/libltfat/single/ifilterbank.o obj/libltfat/single/heapint.o obj/libltfat/single/heap.o obj/libltfat/single/wfacreal.o obj/libltfat/single/idgtreal_long.o obj/libltfat/single/idgtreal_fb.o obj/libltfat/single/iwfacreal.o obj/libltfat/single/pfilt.o obj/libltfat/single/reassign_ti.o obj/libltfat/single/windows.o obj/libltfat/single/dgt_shearola.o obj/libltfat/single/utils.o obj/libltfat/single/rtdgtreal.o obj/libltfat/single/circularbuf.o obj/libltfat/single/slicingbuf.o obj/libltfat/single/dgtrealwrapper.o obj/libltfat/single/dgtrealmp.o obj/libltfat/single/dgtrealmp_parbuf.o obj/libltfat/single/dgtrealmp_kernel.o obj/libltfat/single/dgtrealmp_guts.o obj/libltfat/single/maxtree.o obj/libltfat/single/slidgtrealmp.o obj/libltfat/single/filterbankphaseret.o obj/libltfat/single/fbheapint.o obj/libltfat/single/fftw_wrappers.o obj/libltfat/single/ltfat_blaslapack.o obj/libltfat/single/gabdual_fac.o obj/libltfat/single/gabtight_fac.o obj/libltfat/single/ci_utils.o obj/libltfat/single/ci_windows.o obj/libltfat/single/spread.o obj/libltfat/single/wavelets.o obj/libltfat/single/goertzel.o obj/libltfat/single/reassign.o obj/libltfat/single/gabdual_painless.o obj/libltfat/single/wfac.o obj/libltfat/single/iwfac.o obj/libltfat/single/dgt_long.o obj/libltfat/single/idgt_long.o obj/libltfat/single/dgt_fb.o obj/libltfat/single/idgt_fb.o obj/libltfat/single/ci_memalloc.o obj/libltfat/single/dgtwrapper.o obj/libltfat/single/dct.o obj/libltfat/single/dst.o obj/libltfat/single/gabdual.o obj/libltfat/single/gabtight.o obj/libltfat/complexsingle/ci_utils.o obj/libltfat/complexsingle/ci_windows.o obj/libltfat/complexsingle/spread.o obj/libltfat/complexsingle/wavelets.o obj/libltfat/complexsingle/goertzel.o obj/libltfat/complexsingle/reassign.o obj/libltfat/complexsingle/gabdual_painless.o obj/libltfat/complexsingle/wfac.o obj/libltfat/complexsingle/iwfac.o obj/libltfat/complexsingle/dgt_long.o obj/libltfat/complexsingle/idgt_long.o obj/libltfat/complexsingle/dgt_fb.o obj/libltfat/complexsingle/idgt_fb.o obj/libltfat/complexsingle/ci_memalloc.o obj/libltfat/complexsingle/dgtwrapper.o obj/libltfat/complexsingle/dct.o obj/libltfat/complexsingle/dst.o obj/libltfat/complexsingle/gabdual.o obj/libltfat/complexsingle/gabtight.o
ranlib build/libltfat.a
make[1]: Leaving directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/src'
# Copy to the common place.
mv build/libltfat.a ../lib/
# Compile oct and mex files
make -C ../oct -f Makefile_unix 
make[1]: Entering directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/oct'
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_atrousfilterbank_td.cc
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_cellcoef2tf.cc
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_chirpzt.cc
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_col2diag.cc
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_dct.cc
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_dst.cc
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_dwilt.cc
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_dwiltiii.cc
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_filterbank_fft.cc
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_filterbank_fftbl.cc
make[1]: Leaving directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/thirdparty/polyboolclipper'
mv ../thirdparty/polyboolclipper/polyboolmex.mex .
/nix/store/4l6pc4n5was787p54cz83fc8d58dssij-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_filterbank_td.cc
make[1]: Leaving directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/oct'
make: Leaving directory '/nix/store/bg9f8b9jcvh0fpcabwrqmah84xak887m-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/src'

error: pkg: error running 'make' for the ltfat package
error: called from
    configure_make at line 117 column 9
    build at line 79 column 7
    pkg at line 744 column 7

```

</details>

## Additional context

<!-- Add any other context about the problem here. -->

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->

```
 - system: `"aarch64-darwin"`
 - host os: `Darwin 23.6.0, macOS 14.7.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.24.11`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/store/9bdl6wa2qck3q0f7l892plk5lwy3pvx1-source`
```

## Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@KarlJoad 

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

Ok thanks, that makes sense. I had to add parens, but I can confirm that
```nix
let
  pkgs = import (fetchTarball "https://github.com/KarlJoad/nixpkgs/archive/octavePackages.ltfat/fix-ld-error.tar.gz") {
    config.packageOverrides = pkgs: {
      arpack = pkgs.arpack.overrideAttrs (_: { doCheck = false; });
    };
  };
in
pkgs.mkShell {
  buildInputs = with pkgs; [
    (octave.withPackages(ps: with ps; [ ltfat ]))
  ];
}
```

still results in `ld: library not found for -loctinterp`.

I don't see a difference, but here's the full log when using your branch, for posterity:

<details>

```
Sourcing write-required-octave-packages-hook.sh
Using writeRequiredOctavePackagesPhase
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/4vmpg7vfzwx1xz2vi2m118m177nnjs74-ltfat-2.6.0-of.tar.gz
source root is ltfat
setting SOURCE_DATE_EPOCH to timestamp 1694210710 of file ltfat/inst/demos/demo_dgt_parametrize.m
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
ar: [0;1;35mwarning: [0mcreating build/libltfat.a
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
mkoctfile: stripping disabled on this platform
ld: library not found for -loctinterp
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
make[1]: *** [Makefile_unix:50: comp_dwilt.oct] Error 1
make[1]: *** Waiting for unfinished jobs....
ld: library not found for -loctinterp
ld: library not found for -loctinterp
ld: library not found for -loctinterp
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
make[1]: *** [Makefile_unix:50: comp_col2diag.oct] Error 1
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_atrousfilterbank_td.oct] Error 1
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_dct.oct] Error 1
make[1]: *** [Makefile_unix:50: comp_cellcoef2tf.oct] Error 1
make[1]: *** [Makefile_unix:50: comp_dst.oct] Error 1
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_chirpzt.oct] Error 1
make[1]: *** [Makefile_unix:50: comp_filterbank_fft.oct] Error 1
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
ld: library not found for -loctinterp
make[1]: *** [Makefile_unix:50: comp_filterbank_fftbl.oct] Error 1
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
make[1]: *** [Makefile_unix:50: comp_filterbank_td.oct] Error 1
ld: library not found for -loctinterp
clang++: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
make[1]: *** [Makefile_unix:50: comp_dwiltiii.oct] Error 1
make: *** [Makefile:41: oct] Error 2
make: Entering directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/src'
# Compile the backend lib, no dependency is resolved here since we compile
make -C ../thirdparty/Playrec -f Makefile_unixoct
make -C ../thirdparty/polyboolclipper -f Makefile_unixoct
# a static library
make[1]: Entering directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/thirdparty/Playrec'
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -Wall -mex mex_dll_core.c pa_dll_playrec.c ltfatresample.c \
             -L. -lportaudio -I. -DIS_OCTAVE -DHAVE_PORTAUDIO	-o playrec.mex
make -f Makefile_libltfat build/libltfat.a OPTCFLAGS=-DLTFAT_LARGEARRAYS
make[1]: Entering directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/thirdparty/polyboolclipper'
make[1]: Entering directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/src'
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/memalloc.c -o obj/libltfat/common/dmemalloc.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/error.c -o obj/libltfat/common/derror.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/version.c -o obj/libltfat/common/dversion.o 
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -Wall -mex polyboolmex.cpp clipper.cpp
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/argchecks.c -o obj/libltfat/common/dargchecks.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/dgtwrapper_typeconstant.c -o obj/libltfat/common/ddgtwrapper_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/dgtrealmp_typeconstant.c -o obj/libltfat/common/ddgtrealmp_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/reassign_typeconstant.c -o obj/libltfat/common/dreassign_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/wavelets_typeconstant.c -o obj/libltfat/common/dwavelets_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/integer_manip.c -o obj/libltfat/common/dinteger_manip.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -c modules/libltfat/src/firwin_typeconstant.c -o obj/libltfat/common/dfirwin_typeconstant.o 
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt.c -o obj/libltfat/double/dgt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtreal_fb.c -o obj/libltfat/double/dgtreal_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_multi.c -o obj/libltfat/double/dgt_multi.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_ola.c -o obj/libltfat/double/dgt_ola.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_shear.c -o obj/libltfat/double/dgt_shear.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtreal_long.c -o obj/libltfat/double/dgtreal_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dwilt.c -o obj/libltfat/double/dwilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idwilt.c -o obj/libltfat/double/idwilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/wmdct.c -o obj/libltfat/double/wmdct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/iwmdct.c -o obj/libltfat/double/iwmdct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/filterbank.c -o obj/libltfat/double/filterbank.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ifilterbank.c -o obj/libltfat/double/ifilterbank.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/heapint.c -o obj/libltfat/double/heapint.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/heap.c -o obj/libltfat/double/heap.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/wfacreal.c -o obj/libltfat/double/wfacreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idgtreal_long.c -o obj/libltfat/double/idgtreal_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idgtreal_fb.c -o obj/libltfat/double/idgtreal_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/iwfacreal.c -o obj/libltfat/double/iwfacreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/pfilt.c -o obj/libltfat/double/pfilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/reassign_ti.c -o obj/libltfat/double/reassign_ti.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/windows.c -o obj/libltfat/double/windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_shearola.c -o obj/libltfat/double/dgt_shearola.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/utils.c -o obj/libltfat/double/utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/rtdgtreal.c -o obj/libltfat/double/rtdgtreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/circularbuf.c -o obj/libltfat/double/circularbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/slicingbuf.c -o obj/libltfat/double/slicingbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealwrapper.c -o obj/libltfat/double/dgtrealwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealmp.c -o obj/libltfat/double/dgtrealmp.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealmp_parbuf.c -o obj/libltfat/double/dgtrealmp_parbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealmp_kernel.c -o obj/libltfat/double/dgtrealmp_kernel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtrealmp_guts.c -o obj/libltfat/double/dgtrealmp_guts.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/maxtree.c -o obj/libltfat/double/maxtree.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/slidgtrealmp.c -o obj/libltfat/double/slidgtrealmp.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/filterbankphaseret.c -o obj/libltfat/double/filterbankphaseret.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/fbheapint.c -o obj/libltfat/double/fbheapint.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/fftw_wrappers.c -o obj/libltfat/double/fftw_wrappers.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ltfat_blaslapack.c -o obj/libltfat/double/ltfat_blaslapack.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabdual_fac.c -o obj/libltfat/double/gabdual_fac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabtight_fac.c -o obj/libltfat/double/gabtight_fac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ci_utils.c -o obj/libltfat/double/ci_utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ci_windows.c -o obj/libltfat/double/ci_windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/spread.c -o obj/libltfat/double/spread.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/wavelets.c -o obj/libltfat/double/wavelets.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/goertzel.c -o obj/libltfat/double/goertzel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/reassign.c -o obj/libltfat/double/reassign.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabdual_painless.c -o obj/libltfat/double/gabdual_painless.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/wfac.c -o obj/libltfat/double/wfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/iwfac.c -o obj/libltfat/double/iwfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_long.c -o obj/libltfat/double/dgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idgt_long.c -o obj/libltfat/double/idgt_long.o
make[1]: Leaving directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/thirdparty/Playrec'
mv ../thirdparty/Playrec/playrec.mex .
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgt_fb.c -o obj/libltfat/double/dgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/idgt_fb.c -o obj/libltfat/double/idgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/ci_memalloc.c -o obj/libltfat/double/ci_memalloc.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dgtwrapper.c -o obj/libltfat/double/dgtwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dct.c -o obj/libltfat/double/dct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/dst.c -o obj/libltfat/double/dst.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabdual.c -o obj/libltfat/double/gabdual.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE  -c modules/libltfat/src/gabtight.c -o obj/libltfat/double/gabtight.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_utils.c -o obj/libltfat/complexdouble/ci_utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_windows.c -o obj/libltfat/complexdouble/ci_windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/spread.c -o obj/libltfat/complexdouble/spread.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/wavelets.c -o obj/libltfat/complexdouble/wavelets.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/goertzel.c -o obj/libltfat/complexdouble/goertzel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/reassign.c -o obj/libltfat/complexdouble/reassign.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabdual_painless.c -o obj/libltfat/complexdouble/gabdual_painless.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/wfac.c -o obj/libltfat/complexdouble/wfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/iwfac.c -o obj/libltfat/complexdouble/iwfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgt_long.c -o obj/libltfat/complexdouble/dgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/idgt_long.c -o obj/libltfat/complexdouble/idgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgt_fb.c -o obj/libltfat/complexdouble/dgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/idgt_fb.c -o obj/libltfat/complexdouble/idgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_memalloc.c -o obj/libltfat/complexdouble/ci_memalloc.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgtwrapper.c -o obj/libltfat/complexdouble/dgtwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dct.c -o obj/libltfat/complexdouble/dct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dst.c -o obj/libltfat/complexdouble/dst.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabdual.c -o obj/libltfat/complexdouble/gabdual.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_DOUBLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabtight.c -o obj/libltfat/complexdouble/gabtight.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt.c -o obj/libltfat/single/dgt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtreal_fb.c -o obj/libltfat/single/dgtreal_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_multi.c -o obj/libltfat/single/dgt_multi.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_ola.c -o obj/libltfat/single/dgt_ola.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_shear.c -o obj/libltfat/single/dgt_shear.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtreal_long.c -o obj/libltfat/single/dgtreal_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dwilt.c -o obj/libltfat/single/dwilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idwilt.c -o obj/libltfat/single/idwilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/wmdct.c -o obj/libltfat/single/wmdct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/iwmdct.c -o obj/libltfat/single/iwmdct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/filterbank.c -o obj/libltfat/single/filterbank.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ifilterbank.c -o obj/libltfat/single/ifilterbank.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/heapint.c -o obj/libltfat/single/heapint.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/heap.c -o obj/libltfat/single/heap.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/wfacreal.c -o obj/libltfat/single/wfacreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idgtreal_long.c -o obj/libltfat/single/idgtreal_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idgtreal_fb.c -o obj/libltfat/single/idgtreal_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/iwfacreal.c -o obj/libltfat/single/iwfacreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/pfilt.c -o obj/libltfat/single/pfilt.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/reassign_ti.c -o obj/libltfat/single/reassign_ti.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/windows.c -o obj/libltfat/single/windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_shearola.c -o obj/libltfat/single/dgt_shearola.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/utils.c -o obj/libltfat/single/utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/rtdgtreal.c -o obj/libltfat/single/rtdgtreal.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/circularbuf.c -o obj/libltfat/single/circularbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/slicingbuf.c -o obj/libltfat/single/slicingbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealwrapper.c -o obj/libltfat/single/dgtrealwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealmp.c -o obj/libltfat/single/dgtrealmp.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealmp_parbuf.c -o obj/libltfat/single/dgtrealmp_parbuf.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealmp_kernel.c -o obj/libltfat/single/dgtrealmp_kernel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtrealmp_guts.c -o obj/libltfat/single/dgtrealmp_guts.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/maxtree.c -o obj/libltfat/single/maxtree.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/slidgtrealmp.c -o obj/libltfat/single/slidgtrealmp.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/filterbankphaseret.c -o obj/libltfat/single/filterbankphaseret.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/fbheapint.c -o obj/libltfat/single/fbheapint.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/fftw_wrappers.c -o obj/libltfat/single/fftw_wrappers.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ltfat_blaslapack.c -o obj/libltfat/single/ltfat_blaslapack.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabdual_fac.c -o obj/libltfat/single/gabdual_fac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabtight_fac.c -o obj/libltfat/single/gabtight_fac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ci_utils.c -o obj/libltfat/single/ci_utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ci_windows.c -o obj/libltfat/single/ci_windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/spread.c -o obj/libltfat/single/spread.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/wavelets.c -o obj/libltfat/single/wavelets.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/goertzel.c -o obj/libltfat/single/goertzel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/reassign.c -o obj/libltfat/single/reassign.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabdual_painless.c -o obj/libltfat/single/gabdual_painless.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/wfac.c -o obj/libltfat/single/wfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/iwfac.c -o obj/libltfat/single/iwfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_long.c -o obj/libltfat/single/dgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idgt_long.c -o obj/libltfat/single/idgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgt_fb.c -o obj/libltfat/single/dgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/idgt_fb.c -o obj/libltfat/single/idgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/ci_memalloc.c -o obj/libltfat/single/ci_memalloc.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dgtwrapper.c -o obj/libltfat/single/dgtwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dct.c -o obj/libltfat/single/dct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/dst.c -o obj/libltfat/single/dst.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabdual.c -o obj/libltfat/single/gabdual.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE  -c modules/libltfat/src/gabtight.c -o obj/libltfat/single/gabtight.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_utils.c -o obj/libltfat/complexsingle/ci_utils.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_windows.c -o obj/libltfat/complexsingle/ci_windows.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/spread.c -o obj/libltfat/complexsingle/spread.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/wavelets.c -o obj/libltfat/complexsingle/wavelets.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/goertzel.c -o obj/libltfat/complexsingle/goertzel.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/reassign.c -o obj/libltfat/complexsingle/reassign.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabdual_painless.c -o obj/libltfat/complexsingle/gabdual_painless.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/wfac.c -o obj/libltfat/complexsingle/wfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/iwfac.c -o obj/libltfat/complexsingle/iwfac.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgt_long.c -o obj/libltfat/complexsingle/dgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/idgt_long.c -o obj/libltfat/complexsingle/idgt_long.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgt_fb.c -o obj/libltfat/complexsingle/dgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/idgt_fb.c -o obj/libltfat/complexsingle/idgt_fb.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/ci_memalloc.c -o obj/libltfat/complexsingle/ci_memalloc.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dgtwrapper.c -o obj/libltfat/complexsingle/dgtwrapper.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dct.c -o obj/libltfat/complexsingle/dct.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/dst.c -o obj/libltfat/complexsingle/dst.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabdual.c -o obj/libltfat/complexsingle/gabdual.o
clang -Wall -Wextra -std=c99 -O2 -DNDEBUG -fPIC -Imodules/libltfat/include -DFFTW  -DLTFAT_LARGEARRAYS -DLTFAT_SINGLE -DLTFAT_COMPLEXTYPE -c modules/libltfat/src/gabtight.c -o obj/libltfat/complexsingle/gabtight.o
ar rv build/libltfat.a obj/libltfat/common/dmemalloc.o obj/libltfat/common/derror.o obj/libltfat/common/dversion.o obj/libltfat/common/dargchecks.o obj/libltfat/common/ddgtwrapper_typeconstant.o obj/libltfat/common/ddgtrealmp_typeconstant.o obj/libltfat/common/dreassign_typeconstant.o obj/libltfat/common/dwavelets_typeconstant.o obj/libltfat/common/dinteger_manip.o obj/libltfat/common/dfirwin_typeconstant.o obj/libltfat/double/dgt.o obj/libltfat/double/dgtreal_fb.o obj/libltfat/double/dgt_multi.o obj/libltfat/double/dgt_ola.o obj/libltfat/double/dgt_shear.o obj/libltfat/double/dgtreal_long.o obj/libltfat/double/dwilt.o obj/libltfat/double/idwilt.o obj/libltfat/double/wmdct.o obj/libltfat/double/iwmdct.o obj/libltfat/double/filterbank.o obj/libltfat/double/ifilterbank.o obj/libltfat/double/heapint.o obj/libltfat/double/heap.o obj/libltfat/double/wfacreal.o obj/libltfat/double/idgtreal_long.o obj/libltfat/double/idgtreal_fb.o obj/libltfat/double/iwfacreal.o obj/libltfat/double/pfilt.o obj/libltfat/double/reassign_ti.o obj/libltfat/double/windows.o obj/libltfat/double/dgt_shearola.o obj/libltfat/double/utils.o obj/libltfat/double/rtdgtreal.o obj/libltfat/double/circularbuf.o obj/libltfat/double/slicingbuf.o obj/libltfat/double/dgtrealwrapper.o obj/libltfat/double/dgtrealmp.o obj/libltfat/double/dgtrealmp_parbuf.o obj/libltfat/double/dgtrealmp_kernel.o obj/libltfat/double/dgtrealmp_guts.o obj/libltfat/double/maxtree.o obj/libltfat/double/slidgtrealmp.o obj/libltfat/double/filterbankphaseret.o obj/libltfat/double/fbheapint.o obj/libltfat/double/fftw_wrappers.o obj/libltfat/double/ltfat_blaslapack.o obj/libltfat/double/gabdual_fac.o obj/libltfat/double/gabtight_fac.o obj/libltfat/double/ci_utils.o obj/libltfat/double/ci_windows.o obj/libltfat/double/spread.o obj/libltfat/double/wavelets.o obj/libltfat/double/goertzel.o obj/libltfat/double/reassign.o obj/libltfat/double/gabdual_painless.o obj/libltfat/double/wfac.o obj/libltfat/double/iwfac.o obj/libltfat/double/dgt_long.o obj/libltfat/double/idgt_long.o obj/libltfat/double/dgt_fb.o obj/libltfat/double/idgt_fb.o obj/libltfat/double/ci_memalloc.o obj/libltfat/double/dgtwrapper.o obj/libltfat/double/dct.o obj/libltfat/double/dst.o obj/libltfat/double/gabdual.o obj/libltfat/double/gabtight.o obj/libltfat/complexdouble/ci_utils.o obj/libltfat/complexdouble/ci_windows.o obj/libltfat/complexdouble/spread.o obj/libltfat/complexdouble/wavelets.o obj/libltfat/complexdouble/goertzel.o obj/libltfat/complexdouble/reassign.o obj/libltfat/complexdouble/gabdual_painless.o obj/libltfat/complexdouble/wfac.o obj/libltfat/complexdouble/iwfac.o obj/libltfat/complexdouble/dgt_long.o obj/libltfat/complexdouble/idgt_long.o obj/libltfat/complexdouble/dgt_fb.o obj/libltfat/complexdouble/idgt_fb.o obj/libltfat/complexdouble/ci_memalloc.o obj/libltfat/complexdouble/dgtwrapper.o obj/libltfat/complexdouble/dct.o obj/libltfat/complexdouble/dst.o obj/libltfat/complexdouble/gabdual.o obj/libltfat/complexdouble/gabtight.o obj/libltfat/single/dgt.o obj/libltfat/single/dgtreal_fb.o obj/libltfat/single/dgt_multi.o obj/libltfat/single/dgt_ola.o obj/libltfat/single/dgt_shear.o obj/libltfat/single/dgtreal_long.o obj/libltfat/single/dwilt.o obj/libltfat/single/idwilt.o obj/libltfat/single/wmdct.o obj/libltfat/single/iwmdct.o obj/libltfat/single/filterbank.o obj/libltfat/single/ifilterbank.o obj/libltfat/single/heapint.o obj/libltfat/single/heap.o obj/libltfat/single/wfacreal.o obj/libltfat/single/idgtreal_long.o obj/libltfat/single/idgtreal_fb.o obj/libltfat/single/iwfacreal.o obj/libltfat/single/pfilt.o obj/libltfat/single/reassign_ti.o obj/libltfat/single/windows.o obj/libltfat/single/dgt_shearola.o obj/libltfat/single/utils.o obj/libltfat/single/rtdgtreal.o obj/libltfat/single/circularbuf.o obj/libltfat/single/slicingbuf.o obj/libltfat/single/dgtrealwrapper.o obj/libltfat/single/dgtrealmp.o obj/libltfat/single/dgtrealmp_parbuf.o obj/libltfat/single/dgtrealmp_kernel.o obj/libltfat/single/dgtrealmp_guts.o obj/libltfat/single/maxtree.o obj/libltfat/single/slidgtrealmp.o obj/libltfat/single/filterbankphaseret.o obj/libltfat/single/fbheapint.o obj/libltfat/single/fftw_wrappers.o obj/libltfat/single/ltfat_blaslapack.o obj/libltfat/single/gabdual_fac.o obj/libltfat/single/gabtight_fac.o obj/libltfat/single/ci_utils.o obj/libltfat/single/ci_windows.o obj/libltfat/single/spread.o obj/libltfat/single/wavelets.o obj/libltfat/single/goertzel.o obj/libltfat/single/reassign.o obj/libltfat/single/gabdual_painless.o obj/libltfat/single/wfac.o obj/libltfat/single/iwfac.o obj/libltfat/single/dgt_long.o obj/libltfat/single/idgt_long.o obj/libltfat/single/dgt_fb.o obj/libltfat/single/idgt_fb.o obj/libltfat/single/ci_memalloc.o obj/libltfat/single/dgtwrapper.o obj/libltfat/single/dct.o obj/libltfat/single/dst.o obj/libltfat/single/gabdual.o obj/libltfat/single/gabtight.o obj/libltfat/complexsingle/ci_utils.o obj/libltfat/complexsingle/ci_windows.o obj/libltfat/complexsingle/spread.o obj/libltfat/complexsingle/wavelets.o obj/libltfat/complexsingle/goertzel.o obj/libltfat/complexsingle/reassign.o obj/libltfat/complexsingle/gabdual_painless.o obj/libltfat/complexsingle/wfac.o obj/libltfat/complexsingle/iwfac.o obj/libltfat/complexsingle/dgt_long.o obj/libltfat/complexsingle/idgt_long.o obj/libltfat/complexsingle/dgt_fb.o obj/libltfat/complexsingle/idgt_fb.o obj/libltfat/complexsingle/ci_memalloc.o obj/libltfat/complexsingle/dgtwrapper.o obj/libltfat/complexsingle/dct.o obj/libltfat/complexsingle/dst.o obj/libltfat/complexsingle/gabdual.o obj/libltfat/complexsingle/gabtight.o
ranlib build/libltfat.a
make[1]: Leaving directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/src'
# Copy to the common place.
mv build/libltfat.a ../lib/
# Compile oct and mex files
make -C ../oct -f Makefile_unix 
make[1]: Entering directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/oct'
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_atrousfilterbank_td.cc
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_cellcoef2tf.cc
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_chirpzt.cc
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_col2diag.cc
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_dct.cc
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_dst.cc
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_dwilt.cc
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_dwiltiii.cc
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_filterbank_fft.cc
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_filterbank_fftbl.cc
make[1]: Leaving directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/thirdparty/polyboolclipper'
mv ../thirdparty/polyboolclipper/polyboolmex.mex .
/nix/store/5c19k7g5kp47m17g9anx7s2sdjh6x5ss-octave-9.3.0/bin/mkoctfile-9.3.0 -strip -I../src/modules/libltfat/include -DNDEBUG -L../lib -lltfat comp_filterbank_td.cc
make[1]: Leaving directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/oct'
make: Leaving directory '/nix/store/vy4wb0v7pp16zkrc8vsg4hsnsjlw0ylc-octave-9.3.0-ltfat-2.6.0/ltfat-2.6.0/src'

error: pkg: error running 'make' for the ltfat package
error: called from
    configure_make at line 117 column 9
    build at line 79 column 7
    pkg at line 744 column 7

```
</details>

--- a/pkgs/development/interpreters/octave/default.nix
+++ b/pkgs/development/interpreters/octave/default.nix
@@ -183,7 +183,6 @@ stdenv.mkDerivation (finalAttrs: {
       "--with-lapack=lapack"
       (if use64BitIdx then "--enable-64" else "--disable-64")
     ]
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [ "--enable-link-all-dependencies" ]
     ++ lib.optionals enableReadline [ "--enable-readline" ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [ "--with-x=no" ]
     ++ lib.optionals enableQt [ "--with-qt=5" ];

